sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/SearchAsyncClient.java

```diff
@@ -769,4 +779,18 @@ private static AutocompleteRequest createAutoCompleteRequest(String searchText,
             .setDocument(d)));
         return batch;
     }
+
+    private static class SearchObjectMapper {
+        private static ObjectMapper MAPPER = null;
```

Comment:
Given that this `ObjectMapper` is static and will be shared across instances I don't see an issue with removing the inner class and making this a property on `SearchAsyncClient`.

Reply:
Spotbugs complains the annonymous inner class. I am also flexible to suppress spotbugs and have it in property.